<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkStructBinFile</class>
 <widget class="QDialog" name="WorkStructBinFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>391</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ПРОСМОТРЕТЬ</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>452</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Население:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit_7"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit_8"/>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit_9"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEdit_10"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="textEdit_11"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Площадь города:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Первая буква страны:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Является ли столицей:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Название 5 улиц:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Кол-во домов на улицах:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="textEdit_12"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="inputFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать в файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Содержимое файла:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
